### ISSUE TITLE
Horizontal Hue

### ISSUE THREAD

**Issue by hughanderson4** [superguigui/simple-color-picker#30]
I had a requirement for a horizontal hue selector.
I attempted using CSS transform, but the mouse drag didn't work.
Any way we can have an api option for that?

**Comment by superguigui** [comment]
Yes beside making changes to the CSS you would have to update the logic a bit.
I don't have time nor the need to implement this feature but I'm open for a Pull Request though.
If you want to have a look at it you would have to change things in all the hue related functions :

https://github.com/superguigui/simple-color-picker/blob/master/src/ColorPicker.ts#L311-L327
https://github.com/superguigui/simple-color-picker/blob/master/src/ColorPicker.ts#L370-L396
https://github.com/superguigui/simple-color-picker/blob/master/src/ColorPicker.ts#L177
https://github.com/superguigui/simple-color-picker/blob/master/src/ColorPicker.ts#L153-L154
